index.html

```diff
@@ -3,18 +3,47 @@
 
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="description"
-  content="full description of the app">
-  <meta name="robot" content="index,follow">
-  <link rel="stylesheet" href="styles/reset.css">
-  <link rel="stylesheet" href="styles/debug.css">
-  <title>Document | Description</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
```

Comment:
- [ ] I believe it would be better if you change your title to a more descriptive one. ♻️ 🤔